sdk/servicebus/azure-servicebus/README.md

```diff
@@ -325,6 +338,7 @@ import os
 connstr = os.environ['SERVICE_BUS_CONN_STR']
 queue_name = os.environ['SERVICE_BUS_QUEUE_NAME']
 queue_name = os.environ['SERVICE_BUS_SESSION_ID']
```

Comment:
we already had a session_id os environ, we just called it "queue_name"; let's adjust that rather than adding a new line